packages/material-ui/src/styles/useThemeProps.js

```diff
@@ -1,11 +1,10 @@
 import { getThemeProps } from '@material-ui/styles';
 import useTheme from './useTheme';
-import defaultTheme from './defaultTheme';
 
 export default function useThemeProps({ props: inputProps, name }) {
   const props = { ...inputProps };
 
-  const contextTheme = useTheme() || defaultTheme;
+  const contextTheme = useTheme();
```

Comment:
It looks like a fix that could go into its own pull request

Reply:
I agree with this maybe this is too much, probably gonna just migrate useThemeProps and getThemeProps to typescript, because of typings and functionally it provides mismatch. Gonna close this request.